src/status_im/ui/screens/home/styles.cljs

```diff
@@ -164,3 +166,20 @@
    :width        14
    :height       9
    :tint-color   :white})
+
+(def delete-icon
+  {:color colors/red})
+
+(def delete-icon-highlight
+  {:position         :absolute
+   :top              0
+   :bottom           0
+   :right            -800
+   :width            800
+   :background-color component.styles/color-red-3})
```

Comment:
Is that a different red?

Reply:
This is a background color which show in issue #3013 (as screenshot), which is light red.

Reply:
Please introduce this color in `colors` ns as `light-red`